classes/State.php

```diff
@@ -232,18 +232,6 @@ public static function getStatesByIdCountry($idCountry, $active = false, $orderB
         return Db::getInstance()->executeS($sql);
     }
 
-    /**
-     * Has Counties.
-     *
-     * @param int $idState
-     *
-     * @return int
-     */
-    public static function hasCounties($idState)
```

Comment:
Love it. Yet another method that wasn't possible to use at all 😂 nice!

Reply:
:+1: